src/services/travelServices.js

```diff
@@ -52,7 +52,53 @@ export const showManagerPendingAppr = async (manager) => {
   }
 };
 
-export const showUsertravelsStatus = async (userId) => {
+const checkApprovalStatus = async (id, userId) => {
+  try {
```

Comment:
The `findTravelById` function above should suffice for this function, and it even does more. I will suggest you use it, as this function seems redundant.

Reply:
@chuksjoe  Good point

However, That function does more than find the request. It ensures that the requester is the modifier, so it is not redundant. Its an important part of the feature.